### PR TITLE
fixup! Add parser support for the OVER clause to function calls

### PR DESCRIPTION
**Pull request by andreidan**
## Checklist

 - [x] User relevant changes are recorded in ``CHANGES.txt``
 - [x] Touched code is covered by tests
 - [x] Documentation has been updated if necessary
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: sql-parser/src/main/java/io/crate/sql/SqlFormatter.java [hunk 1]
```diff
@@ -817,6 +817,17 @@ static String formatSortItem(SortItem sortItem) {
...
+
+        switch (sortItem.getNullOrdering()) {
+            case FIRST:
+                sb.append(" NULLS FIRST");
+                break;
+            case LAST:
+                sb.append(" NULLS LAST");
+                break;
+            default:
+                break;
```
Comment (mfussenegger) [inline sql-parser/src/main/java/io/crate/sql/SqlFormatter.java]: Should we add an assertion error here instead?

Comment (andreidan) [reply sql-parser/src/main/java/io/crate/sql/SqlFormatter.java]: as this is the sql formatter, I'd say empty/no result is a valid option

Comment (andreidan) [reply sql-parser/src/main/java/io/crate/sql/SqlFormatter.java]: or do you think we should print out the default if nothing is specified? (we actually expect this option to not be defined as the SortItem has an `UNDEFINED` null ordering)

Comment (mfussenegger) [reply sql-parser/src/main/java/io/crate/sql/SqlFormatter.java]: Ah right, didn't think of the `UNDEFINED`. One option would be to make the `UNDEFINED` case explicit and add a default assertion case. But it is fine as it is as well.

**Pull request merged**